pkg/relayer/cmd/cmd.go

```diff
@@ -66,7 +102,8 @@ func runRelayer(cmd *cobra.Command, _ []string) error {
 	// Handle interrupt and kill signals asynchronously.
 	signals.GoOnExitSignal(cancelCtx)
 
-	configContent, err := os.ReadFile(flagRelayMinerConfig)
+	// TODO: come back to this... needs to be overridden by flags...
```

Comment:
1. TODO_IN_THIS_COMMIT?
2. If we set up the defaults correctly above, shouldn't it be what we expect?

Reply:
This is being done in the function that parses the config at the moment. It's not automatic because viper's API won't provide a deserialized data structure from the config (which is what we want). Instead it will parse the config and make everything available via the `viper.Get...` API, which we can use to override the sensitized config struct.